vault/request_handling.go

```diff
@@ -561,7 +561,7 @@ func (c *Core) handleCancelableRequest(ctx context.Context, req *logical.Request
 			// be revoked after the call. So we have to do the validation here.
 			valid, err := c.validateWrappingToken(ctx, req)
 			if err != nil {
-				return nil, fmt.Errorf("error validating wrapping token: %w", err)
+				return logical.ErrorResponse(fmt.Errorf("error validating wrapping token: %w", err).Error()), logical.ErrPermissionDenied
```

Comment:
Do we need to worry about constructing an error/wrapping the error here, since we're just calling `.Error()` to get the string out of it?

Reply:
Nope, that's a good point. I'll change that.